### PR TITLE
chore: fonts cleanup

### PR DESCRIPTION
**Pull request by nicolasburtey**
before:
-rw-r--r--@ 1 n  staff  41977671 Feb 16 15:15 app-arm64-v8a-release.apk
-rw-r--r--@ 1 n  staff  35179554 Feb 16 15:15 app-armeabi-v7a-release.apk
-rw-r--r--@ 1 n  staff  99688871 Feb 16 15:15 app-universal-release.apk
-rw-r--r--@ 1 n  staff  43504625 Feb 16 15:15 app-x86-release.apk
-rw-r--r--@ 1 n  staff  43940437 Feb 16 15:15 app-x86_64-release.apk
-rw-r--r--@ 1 n  staff      1484 Feb 16 15:15 output-metadata.json

after (first commit)
-rw-r--r--@ 1 n  staff  38779778 Feb 16 15:17 app-arm64-v8a-release.apk
-rw-r--r--@ 1 n  staff  31981661 Feb 16 15:17 app-armeabi-v7a-release.apk
-rw-r--r--@ 1 n  staff  96490911 Feb 16 15:17 app-universal-release.apk
-rw-r--r--@ 1 n  staff  40306730 Feb 16 15:17 app-x86-release.apk
-rw-r--r--@ 1 n  staff  40742526 Feb 16 15:17 app-x86_64-release.apk
-rw-r--r--@ 1 n  staff      1484 Feb 16 15:17 output-metadata.json

3rd commits:
-rw-r--r--@ 1 n  staff  36956227 Feb 16 18:38 app-arm64-v8a-release.apk
-rw-r--r--@ 1 n  staff  30158105 Feb 16 18:38 app-armeabi-v7a-release.apk
-rw-r--r--@ 1 n  staff  94667311 Feb 16 18:38 app-universal-release.apk
-rw-r--r--@ 1 n  staff  38483186 Feb 16 18:38 app-x86-release.apk
-rw-r--r--@ 1 n  staff  38918986 Feb 16 18:38 app-x86_64-release.apk
-rw-r--r--@ 1 n  staff      1484 Feb 16 18:38 output-metadata.json

### CODE REVIEW

**Review by Extheoisah — APPROVED** [review]

File: ios/GaloyApp/Info.plist [hunk 1]
```diff
@@ -101,64 +101,10 @@
...
-		<string>FontAwesome5_Regular.ttf</string>
-		<string>FontAwesome5_Solid.ttf</string>
-		<string>Foundation.ttf</string>
 		<string>Ionicons.ttf</string>
-		<string>MaterialIcons.ttf</string>
-		<string>MaterialCommunityIcons.ttf</string>
-		<string>SimpleLineIcons.ttf</string>
-		<string>Octicons.ttf</string>
-		<string>Zocial.ttf</string>
-		<string>Fontisto.ttf</string>
```
Comment (daviroo) [inline ios/GaloyApp/Info.plist:122]: Looks like we removed all the font's aside from `Ionicons`.  Was there a reason we done this?

Comment (nicolasburtey) [reply ios/GaloyApp/Info.plist:122]: reducing build size

Comment (daviroo) [reply ios/GaloyApp/Info.plist:122]: react native elements uses [react-native-vector-icons](https://reactnativeelements.com/docs/components/icon) library for components.  It requires these native fonts for rendering the icons.  If we don't include them we might have issues when using components from the library.  Should we add them back in?

Comment (nicolasburtey) [reply ios/GaloyApp/Info.plist:122]: build size is important.

we should use icons from `Ionicons` unless the icon is not present there, in that case we can import just one another fonts/set of icons.

Comment (daviroo) [reply ios/GaloyApp/Info.plist:122]: I've raised a PR to add them back in https://github.com/GaloyMoney/galoy-mobile/pull/1184

**Pull request merged**